src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -74,6 +78,7 @@ export const CheckoutDetails = () => {
   const quoteDetails = data.quoteDetails
   const bundleVariants = data.bundleVariants
   const allQuotes = getUniqueQuotesFromVariantList(bundleVariants)
+  const offerPageLink = `/${localePath}/new-member/offer/${quoteCartId}`
```

Comment:
Hmm also here, any reason to define it outside the function that uses this variable?